src/components/Contacts/ContactDetails/ContactNotesTab/ContactNotesTab.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { TextField } from '@material-ui/core';
+
+export const ContactNotesTab: React.FC = () => {
+  return (
+    <TextField
+      fullWidth
+      multiline
+      placeholder="Add contact notes"
```

Comment:
Since this is a constant string, it should be setup for translation into other languages.

Add this to the top
```ts
import { useTranslation } from 'react-i18next';
```
call the `useTranslation` hook in the component to get access to the `t` function
```ts
const { t } = useTranslation();
```
and then call the `t` function with the string to be translated
```suggestion
      placeholder={t('Add contact notes')}
```

Reply:
Nice catch. I did see that translation being used elsewhere. I'll get this in place.